### ISSUE TITLE
Specify what branch to build

### ISSUE THREAD

**Issue by aullman** [coderanger/nightlies#1]
Would it be possible to specify which branch to build? Or to build multiple branches?

**Comment by coderanger** [comment]
Yeah, the code that triggers the build is https://github.com/coderanger/nightlies/blob/master/model.rb#L99 and that just has master hardcoded right now. Would need UI and a place in the DB to stuff it.

**Comment by aullman** [comment]
For now I worked around this issue by just forking a separate version and putting the branch I wanted built as the master branch in that fork :)

**Comment by coderanger** [comment]
:+1: Will leave this open so I at least remember to add the column.

**Comment by soullivaneuh** [comment]
:+1: For multiple branches.

**Comment by floweisshardt** [comment]
+1 for making the branch configurable

**Comment by aioue** [comment]
+1

**Comment by ldionne** [comment]
+1

**Comment by glennmatthews** [comment]
:+1:

**Comment by linuxenko** [comment]
+1

**Comment by ldionne** [comment]
Note that TravisCI now supports cron jobs out of the box.

**Comment by linuxenko** [comment]
@ldionne  no, it does not.  it requires emailing to support, depends on their mood . .

**Comment by ldionne** [comment]
I can go to the `settings` tab of my repos and just enable that there. Maybe I'm special, but I don't remember asking them for anything special.

**Comment by linuxenko** [comment]
aha, looks like you are special.

https://docs.travis-ci.com/user/cron-jobs
